### PR TITLE
[mlvm] adding dask + maintence updates

### PR DESCRIPTION
**Pull request by bradmiro**
- Added `dask` init action
- Updated package versions
- Now uses temporary mamba environment to install conda packages
- Updated README to reflect changes
- Conflict with `r-xgboost` and RAPIDS; will not install the former if the latter is requested via `--metadata rapids-runtime`
- Issues `mxnet` and `rpy2` with conda. Moved to pip, as their respective communities suggest.

### CODE REVIEW

File: integration_tests/dataproc_test_case.py [hunk 1]
```diff
@@ -310,6 +310,8 @@ def run_command(cmd, timeout_in_minutes=DEFAULT_TIMEOUT):
                           "gcloud beta compute scp --internal-ip ") if (
                               INTERNAL_IP_SSH
                               and "gcloud compute scp " in cmd) else cmd
+        cmd = cmd + " -- -o \"ProxyCommand corp-ssh-helper %h %p\"" if "gcloud compute ssh" in cmd else cmd
+        cmd = cmd + " --scp-flag=\"-oProxyCommand corp-ssh-helper %h %p\"" if "gcloud compute scp" in cmd else cmd
```
Comment (medb) [inline integration_tests/dataproc_test_case.py:314]: Why is this necessary?

File: mlvm/BUILD [hunk 2]
```diff
@@ -9,11 +9,14 @@ py_test(
         "scripts/spark_bq.py",        
         "scripts/verify_rapids_dask.py",
         "scripts/verify_rapids_spark.py",
+        "scripts/verify_dask_standalone.py",
+        "scripts/verify_dask_yarn.py",
         "//gpu:install_gpu_driver.sh",
+        "//dask:dask.sh",
         "//rapids:rapids.sh",
```
Comment (medb) [inline mlvm/BUILD:16]: Sort dependencies in alphabetical order

File: mlvm/mlvm.sh [hunk 3]
```diff
@@ -106,23 +113,34 @@ function install_gpu_drivers() {
...
 
 function install_conda_packages() {
+  local base
+  base=$(conda info --base)
+  local -r mamba_env_name=mamba
+  local -r mamba_env=${base}/envs/mamba
   local -r extra_packages="$(/usr/share/google/get_metadata_value attributes/CONDA_PACKAGES || echo "")"
   local -r extra_channels="$(/usr/share/google/get_metadata_value attributes/CONDA_CHANNELS || echo "")"
-
+
```
Comment (medb) [inline mlvm/mlvm.sh:122]: Remove extra space

**Review by medb — CHANGES_REQUESTED** [review]

File: integration_tests/dataproc_test_case.py (hunk 1 above)
Comment (bradmiro) [reply integration_tests/dataproc_test_case.py]: Oops, reverted

File: mlvm/BUILD (hunk 2 above)
Comment (bradmiro) [reply mlvm/BUILD]: Done

File: mlvm/mlvm.sh (hunk 3 above)
Comment (bradmiro) [reply mlvm/mlvm.sh]: Done

**Comment by bradmiro** [comment]
Can we try testing this again?

**Comment by bradmiro** [comment]
The test failed because it ran tests that... it was supposed to skip:
https://github.com/GoogleCloudDataproc/initialization-actions/blob/e103b33cfe5ae5620c8715506bab6241f38dac6b/mlvm/test_mlvm.py#L122

Oddly enough, this worked completely fine locally. Not sure what happened. I also rebased, maybe we should try again?

**Review by medb — CHANGES_REQUESTED** [review]

File: mlvm/README.md [hunk 4]
```diff
@@ -80,7 +86,7 @@ config:
       include-gpus: 'true'
       rapids-runtime: SPARK
   initializationActions:
-  - executableFile: gs://bmiro-test/dataproc-initialization-actions/mlvm/mlvm.sh
+  - executableFile: gs://dataproc-initialization-actions/mlvm/mlvm.sh
```
[the remark this answers is not in the document]
Comment (bradmiro) [reply mlvm/README.md]: Done

File: mlvm/README.md [hunk 5]
```diff
@@ -60,9 +65,10 @@ gcloud dataproc clusters create ${CLUSTER_NAME} \
     --metadata gpu-driver-provider=NVIDIA \
     --metadata rapids-runtime=SPARK \
     --metadata include-gpus=true \
+    --metadata init-actions-repo=${INIT_ACTIONS_REPO} \
     --optional-components JUPYTER \
     --initialization-actions gs://dataproc-initialization-actions/mlvm/mlvm.sh \
```
[the remark this answers is not in the document]
Comment (bradmiro) [reply mlvm/README.md]: Fixed

**Review by medb — APPROVED** [review]

**Pull request merged**